shenyu-common/src/main/java/org/apache/shenyu/common/dto/convert/selector/DubboUpstream.java

```diff
@@ -246,8 +246,6 @@ public boolean equals(final Object o) {
         }
         DubboUpstream that = (DubboUpstream) o;
         return port == that.port
-                && weight == that.weight
-                && warmup == that.warmup
```

Comment:
why remove these ?

Reply:
These two are not the key factors. See `DivideUpstream#equals`